### PR TITLE
More emotes!

### PR DESCRIPTION
**Pull request by MisterLayne**
Adds emotes for _**ERP PURPOSES**_ (or just cause they are some good emotes)

### CODE REVIEW

**Comment by Belsima** [comment]
>no chance for flips to fail and break bones with clumsy mutation

**Comment by Anewbe** [comment]
Hmm... Movement ones are questionable, whistles... Might be nice to have sound files for those?

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -700,6 +700,36 @@
...
+		if("qwhistle")
+			if(!muzzled)
+				message = "whistles quietly."
+			else
+				message = "spits lightly, a failed attempt at a whistle."
+
+		if("spin" || "spins" || "crazy")
+			if(!src.restrained())
+				message = "spins frantically!"
+				spawn(0)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: no

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -700,6 +700,36 @@
...
+					for(var/i in list(1,2,4,8,2,4,1,2,4,8,2,4,1,2,4,8,2,4,1,2,4,8,2,4,))
+						dir = i
+						sleep(1)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: stop

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -700,6 +700,36 @@
...
+			else
+				message = "spits lightly, a failed attempt at a whistle."
+
+		if("qwhistle")
+			if(!muzzled)
+				message = "whistles quietly."
+			else
+				message = "spits lightly, a failed attempt at a whistle."
+
+		if("spin" || "spins" || "crazy")
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: datum emotes when

**Comment by Anewbe** [comment]
Where did those sound files come from? Even if they're free use, we need to credit them, or something.

**Comment by MisterLayne** [comment]
Sound files came from SoundBible.com

**Pull request merged**